addons/binding/org.openhab.binding.yamahareceiver2/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,100 @@
+            <parameter name="HOST" type="text" required="true">
+                <context>network-address</context>
+                <label>Address</label>
+                <description>The address of the AVR to control.</description>
+            </parameter>
+        </config-description>
+        
+    </thing-type>
+
+    <channel-type id="power">
+        <item-type>Switch</item-type>
+        <label>Power</label>
+        <description>Power ON/OFF the AVR</description>
+    </channel-type>
+    
+    <channel-type id="input">
+        <item-type>String</item-type>
+        <label>Input source</label>
+        <description>Select the input source of the AVR</description>
+    </channel-type>
+    
+    <channel-type id="netradiotune">
+        <item-type>Number</item-type>
+        <label>NetRadio Channel</label>
+        <description>Select the net radio channel of the AVR</description>
+    </channel-type>
+    
+    <channel-type id="surroundProgram">
+        <item-type>String</item-type>
+        <label>Surround program</label>
+        <description>Select the surround program of the AVR</description>
+        <state>
+            <options>
+                <option value="Music Video">Music Video</option>
+                <option value="The Roxy Theater">The Roxy Theater</option>
+                <option value="Cellar Club">Cellar Club</option>
+                <option value="Hall in Vienna">Hall in Vienna</option>
+                <option value="Hall in Munich">Hall in Munich</option>
+                <option value="Roleplaying Game">Roleplaying Game</option>
+                <option value="Action Game">Action Game</option>
+                <option value="Sports">Sports</option>
+                <option value="Mono Movie">Mono Movie</option>
+                <option value="Drama">Drama</option>
+                <option value="Adventure">Adventure</option>
+                <option value="Sci-Fi">Sci-Fi</option>
+                <option value="Spectacle">Spectacle</option>
+                <option value="Standard">Standard</option>
+                
+                <option value="Pro Logic">Pro Logic</option>
+                <option value="Neo:6 Music">Neo:6 Music</option>
+                <option value="Neo:6 Cinema">Neo:6 Cinema</option>
+                <option value="PLII[x] Game">PLII[x] Game</option>
+                <option value="PLII[x] Music">PLII[x] Music</option>
+                <option value="PLII[x] Movie">PLII[x] Movie</option>
+                <option value="7ch Enhancer">7ch Enhancer</option>
+
+                <option value="2ch Stereo">2ch Stereo</option>
+                <option value="7ch Stereo">7ch Stereo</option>
+                <option value="STRAIGHT">STRAIGHT</option>
+                <option value="Straight Enhancer">Straight Enhancer</option>
+            </options>
+        </state>
+    </channel-type>
+    
+    <channel-type id="volume">
+        <item-type>Number</item-type>
```

Comment:
You should have the "volume" channel be of "Dimmer" type, which takes 0-100% and which can process INCREASE/DESCREASE commands as well - this is how all other bindings do it as well and which is best supported in the UIs.
If you also want to offer a "Volume in dB" channel additionally, you can do so, but it should be marked as an "advanced" channel.